### DISCUSSION TITLE
Streamfx failing to launch on arch

### DISCUSSION THREAD

**Discussion by Redd56** [Xaymar/obs-stream-effects#872] [General]
### Operating System

Linux (like Arch Linux)

### OBS Studio Version

27.2.4-2

### StreamFX Version

both 11.1 and 12.0a45 beta 

### OBS Studio Log

https://obsproject.com/logs/3FfT4r7L8Bah--Jb

### Current and Expected Behavior

current behavior: streamfx fails to load due to missing libavutil.so.56 (i believe ffmpeg was updated to now be 59 recently hence why it cant find version 56)

expected behavior: streamfx to load so long as ffmpeg version is atleast a certain age (so long as features arent missing)

### Steps to Reproduce the Problem

install obs with the latest arch packages.
open obs with streamfx installed in the plugins directory. 
no streamfx

**Comment by Xaymar** [comment]
#### Rejected: User Error
StreamFX does not officially support Arch Linux, only Ubuntu. Please make sure you've read the [System Requirements](https://github.com/Xaymar/obs-StreamFX/wiki/System-Requirements).

**Reply by Xaymar** [reply]
I converted this into a discussion, as this is not a bug.

**Reply by Redd56** [reply]
fair enough, any ideas on how to fix it not launching due to the newer ffmpeg version?

**Reply by Redd56** [reply]
and now its running? okay thank you ffmpeg

**Reply by Xaymar** [reply]
Either install the Arch Linux packages maintained by a third party, or build from source with the newer FFmpeg binaries.

**Reply by Redd56** [reply]
Well i dont know how but by the grace of package maintainers libavutil is now being detected by streamfx, just did an update yesterday and it magicly started running. my only guess is a single day long flubbed update for the libavutils binary somehow. anyway issue resolved
